merlin/models/tf/prediction_tasks/next_item.py

```diff
@@ -187,13 +220,9 @@ def NextItemPredictionTask(
         )
 
     else:
-        if weight_tying:
-            prediction_call = ItemsPredictionWeightTying(schema)
-
-        else:
-            prediction_call = ItemsPrediction(schema)
+        prediction_call = ItemsPrediction(schema)
```

Comment:
Do we want to remove `if weight_trying is True: prediction_call = ItemsPredictionWeightTying(schema)`?